OpenRA.Mods.Common/Activities/CaptureActor.cs

```diff
@@ -28,6 +28,19 @@ public CaptureActor(Actor self, Target target)
 			manager = self.Trait<CaptureManager>();
 		}
 
+		protected override void TickInner(Actor self, Target target, bool targetIsDeadOrHiddenActor)
+		{
+			if (target.Type == TargetType.Actor && enterActor != target.Actor)
+			{
+				enterActor = target.Actor;
+				enterCaptureManager = target.Actor.TraitOrDefault<CaptureManager>();
+			}
+
+			if (!targetIsDeadOrHiddenActor && target.Type != TargetType.FrozenActor &&
+				(enterCaptureManager == null || !enterCaptureManager.CanBeTargetedBy(enterActor, self, manager)))
+				Cancel(self);
```

Comment:
I believe this should be `Cancel(self, true)`, as per the `TickInner` summary and other `Enter` subclasses.

Reply:
Fixed